lib/frosty_friend/temperature/store.ex

```diff
@@ -18,16 +18,20 @@ defmodule FrostyFriend.Temperature.Store do
   def list(%DateTime{} = date_time, measurement \\ :celsius, time_order \\ :asc) do
     date_comp = DateTime.to_unix(date_time)
 
-    select_pattern = if measurement == :celsius do
-      [{{:"$1", :"$2", :_}, [{:>=, :"$1", date_comp}], [{{:"$1", :"$2"}}]}]
-    else
-      [{{:"$1", :_, :"$2"}, [{:>=, :"$1", date_comp}], [{{:"$1", :"$2"}}]}]
-    end
+    select_pattern =
+      if measurement == :celsius do
+        [{{:"$1", :"$2", :_}, [{:>=, :"$1", date_comp}], [{{:"$1", :"$2"}}]}]
+      else
+        [{{:"$1", :_, :"$2"}, [{:>=, :"$1", date_comp}], [{{:"$1", :"$2"}}]}]
+      end
 
     :temperatures
     |> :ets.select(select_pattern)
-    |> Enum.map(fn {unix_date, temp} -> {DateTime.from_unix!(unix_date), temp} end)
+    |> Enum.map(fn {unix_date, temp} ->
+      {NaiveDateTime.to_iso8601(DateTime.from_unix!(unix_date)), temp}
```

Comment:
I don't remember the reason for NaiveDateTime, should we just keep utc and swap based on browser location?

Reply:
Sounds good, I added it trying to get the CharJs hook to take the data